Source/Core/writeTextToCanvas.js

```diff
@@ -72,32 +157,28 @@ function writeTextToCanvas(text, options) {
   context2D.lineWidth = strokeWidth;
   context2D[imageSmoothingEnabledName] = false;
 
-  // textBaseline needs to be set before the measureText call. It won't work otherwise.
-  // It's magic.
-  context2D.textBaseline = defaultValue(options.textBaseline, "bottom");
-
   // in order for measureText to calculate style, the canvas has to be
   // (temporarily) added to the DOM.
   canvas.style.visibility = "hidden";
   document.body.appendChild(canvas);
 
-  var dimensions = measureText(context2D, text, stroke, fill);
+  var dimensions = measureText(context2D, text, font, stroke, fill);
   canvas.dimensions = dimensions;
```

Comment:
Can `canvas.dimensions = dimensions;` be removed? `HtmlCanvasElement` doesn't have a `dimensions` property and it wouldn't make sense for it to be set to a `TextMetrics` object anyways.

Reply:
Never mind, I see that this property is used by `LabelCollection`. Could you add a comment about that? Otherwise others might get confused.